pythran/cxxgen.py

```diff
@@ -461,9 +461,14 @@ def __init__(self, name, docstrings, metadata):
         self.metadata['moduledoc'] = moduledoc
 
     def docstring(self, doc):
-        return '"%s"' % (dedent(doc).replace('"', '\\"')
-                                    .replace('\n', '\\n')
-                                    .replace('\r', '\\r'))
+        return self.splitstring(dedent(doc).replace('"', '\\"')
+                                           .replace('\n', '\\n')
+                                           .replace('\r', '\\r'))
+
+    def splitstring(self, doc):
+        if len(doc) == 0:
+            return '""'
+        return "".join(['"%s"' % s for s in wrap(doc, 16380)])
```

Comment:
An interesting choice. I wonder if we could take an approach that would prevent this magic (uncommented) number, something along

```
return "".join('"{}\\n"'.format(s) for s in doc.split('\\n'))
```